google/cloud/datastore/query.py

```diff
@@ -540,6 +539,21 @@ def _next_page(self):
         response_pb = self.client._datastore_api.run_query(
             self._query.project, partition_id, read_options, query=query_pb
         )
+
+        while (
```

Comment:
This is showing up in the diff but this is already in master:

https://github.com/googleapis/python-datastore/blob/9a3d3148581fede2e4cc341ff09ee327248df0e7/google/cloud/datastore/query.py#L543-L558